src-ts/tools/learn/course-details/course-curriculum/CourseCurriculum.tsx

```diff
@@ -2,9 +2,16 @@ import { Dispatch, FC, SetStateAction, useCallback, useState } from 'react'
 import { NavigateFunction, useNavigate } from 'react-router-dom'
 
 import { Button } from '../../../../lib'
-import { CourseOutline, LearningHat, TcAcademyPolicyModal } from '../../components'
+import {
+    CourseOutline,
+    LearnCourse,
+    LearningHat,
+    LearnLesson,
+    LearnModule,
+    LearnMyCertificationProgress
+} from '../../learn-lib'
 import { getFccLessonPath } from '../../learn.routes'
-import { LearnCourse, LearnLesson, LearnModule, LearnMyCertificationProgress } from '../../services'
+import { TcAcademyPolicyModal } from '../tc-academy-policy-modal'
```

Comment:
i see now that this modal is only in the course-curriculum, so let's nest its folder

/course-curriculum/tc-academy-policy-modal

Reply:
i wend ahead and changed this myself